docs/src/use/configure/configuration-files.md

```diff
@@ -46,25 +52,6 @@ module.exports = [
 ];
 ```
 
-The configuration file can also export a promise that resolves to the configuration array. This can be useful for using ESM dependencies in CommonJS configuration files, as in this example:
```

Comment:
Perhaps we could leave the first sentence, as that functionality is supported regardless of the purpose?

Reply:
I wasn't sure what kind of an example we could give, which is why I deleted the whole thing. It seems odd to mention that the config file can return a promise but then not give a use case where that's necessary.

Reply:
I still think we should at least mention the functionality even though it might not be necessary for any particular use case as there are other ways, but we can revisit this later.